### DISCUSSION TITLE
Binds : on release

### DISCUSSION THREAD

**Discussion by RichardFevrier** [YaLTeR/niri#569] [Ideas]
Trying to use `SwayOSD` for `Caps_Lock` with `binds` like so:
```
spawn-at-startup "swayosd-server"

binds {
    Caps_Lock { spawn "swayosd-client" "--caps-lock"; }
}
```
I noticed that `Niri` is grabbing the event and so the uppercase is not applied.

I checked for `Sway` and apparently they use a `--release` flag to handle that case.

**Comment by YaLTeR** [comment]
Yeah, you can't bind on release atm. It's not trivial to add
